### PR TITLE
Update PerfCollect to Create PerfMap Files Using Crossgen

### PR DESCRIPTION
**Pull request by Lucrecious**

### CODE REVIEW

**Comment by Lucrecious** [comment]
@brianrob hey, could you review this?

File: src/performance/perfcollect/perfcollect [hunk 1]
```diff
@@ -1077,12 +1078,6 @@ ProcessArguments()
 		then
 			events=$value
 			i=$i+1
-		elif [ "-collectsec" == "$arg" ]
```
Comment (brianrob) [inline src/performance/perfcollect/perfcollect]: It looks like you inadvertently removed the -collectsec functionality.  Can you please put this back?

File: src/performance/perfcollect/perfcollect [hunk 2]
```diff
@@ -1077,12 +1078,6 @@ ProcessArguments()
...
-		then
```
Comment (brianrob) [inline src/performance/perfcollect/perfcollect]: Should this be removed?

File: src/performance/perfcollect/perfcollect [hunk 3]
```diff
@@ -1217,6 +1229,77 @@ ProcessCollectedData()
...
+		crossgenDir=`dirname "${file}"`
+		if [ -f ${crossgenDir}/crossgen ]
+		then
+			crossgenCmd=${crossgenDir}/crossgen
+			echo "Found crossgen at ${crossgenCmd}"
+			break
+		fi
+	done
+
+        OLDIFS=$IFS
```
Comment (brianrob) [inline src/performance/perfcollect/perfcollect]: Can you fix the intending here?  Looks like you have an extra space on a few lines.

File: src/performance/perfcollect/perfcollect [hunk 4]
```diff
@@ -1409,13 +1489,8 @@ BuildPerfRecordArgs()
 
 	done
 
-	if [ ! -z ${duration} ]
```
Comment (brianrob) [inline src/performance/perfcollect/perfcollect]: Looks like more sleep functionality that was removed.

File: src/performance/perfcollect/perfcollect [hunk 5]
```diff
@@ -1463,7 +1540,6 @@ DoCollect()
 			sleep 1
 		done
 	fi
-	EndCollect
```
Comment (brianrob) [inline src/performance/perfcollect/perfcollect]: Same here.  More sleep functionality.  Can you make sure that the diff is just your intended changes?  I'm sure that what happened is that you copied a file into your repo after these updates were made, and so they show up as deletes.

**Comment by Lucrecious** [comment]
@brianrob Hey! I manually added everything (that I accidentally deleted) back in, is that OK? You can look it over.

**Comment by brianrob** [comment]
LGTM

**Pull request merged**